metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -103,6 +103,11 @@ class SnowflakeFilterConfig(SQLFilterConfig):
         description="Regex patterns for streams to filter in ingestion. Note: Defaults to table_pattern if not specified. Specify regex to match the entire view name in database.schema.view format. e.g. to match all views starting with customer in Customer database and public schema, use the regex 'Customer.public.customer.*'",
     )
 
+    procedure_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
```

Comment:
> Defaults to table_pattern if not specified

Maybe you mean: "defaults to _allow all_ and, in case of specifically set to null, table pattern is applied." ?
However, I have not seen the fallback to use table pattern in the code

This requires some refinement